MonstieWash/Assets/Scripts/Consumables/ConsumablesManager.cs

```diff
@@ -15,10 +15,10 @@ public class ConsumablesManager : MonoBehaviour
     [Tooltip("Gameobject new Ui consumable items will be parented under")] [SerializeField] private GameObject imageHolder; //Empty Gameobject in Ui to hold images; (I tried changing to transform as per feedback but doesn't seem to work with rect transform in ui)
     [Tooltip("A template obj requiring, image, UIconsumableScript, button linked to the scripts onclick")] [SerializeField] private GameObject refObject; //blank object for instantiate to reference; Needs a button, UIConsumableScript, Image.
     [Tooltip("Layer that checks player mouse is over monster collider")] [SerializeField] private LayerMask monsterLayer; //Layer of monster hitbox;
-    [Tooltip("Layer that checks player mouse is over monster collider")] [SerializeField] public LayerMask consumableLayer; //Layer of consumables hitbox;
+    [Tooltip("Layer that checks player mouse is over bag to open its contents")] [SerializeField] public LayerMask bagLayer; //Layer of consumables hitbox;
+    [Tooltip("Layer that checks player mouse is over consumable collider")] [SerializeField] public LayerMask itemConsumableLayer; //Layer of consumables hitbox;
 
-    [Header("All consumables go here")]
-    [Tooltip("List all starting consumables")] [SerializeField] private List<consumableData> consumableList;
+    [SerializeField] private Inventory m_inventory; //Reference to Inventory;
```

Comment:
also minor but this doesn't need the `m_` prefix since it's serialized.

```suggestion
    [SerializeField] private Inventory inventory; //Reference to Inventory;
```